gen3/bin/roll-wait.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
```

Comment:
We already have `kube-wait4-pods`.  You can add a `--wait` flag to `roll.sh` rather than introduce a new command